common/Makefile.am

```diff
@@ -206,6 +206,19 @@ endif # ENABLE_DARK
 
 endif #!ENABLE_TRANSPARENCY
 
+	pwd
```

Comment:
Is there some purpose for this single `pwd` that I'm missing? or was it accidentally left in?

Reply:
good catch. removed